src/diffs/package-json.js

```diff
@@ -0,0 +1,37 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('lodash')
```

Comment:
I genuinely feel that lodash can tend to make code more confusing with little to gain. Unless you feel strongly about bringing it in, I'd much rather leave it out.